### PR TITLE
DOC: Add contributing section in README.rst and README.rst in doc/

### PR DESCRIPTION
**Pull request by raybellwaves**
- [NA] Closes #xxxx (remove if there is no corresponding issue, which should only be the case for minor changes)
 - [NA] Tests added (for all bug fixes or enhancements)
 - [NA] Tests passed (for all non-documentation changes)
 - [NA] Fully documented, including `whats-new.rst` for all changes and `api.rst` for new API (remove if this change should not be visible to users, e.g., if it is an internal clean-up, or if this is part of a larger project that will be documented later)

These were inspired by a discussion on the mailing list:
https://groups.google.com/forum/#!topic/xarray/FyCFkuWbFSk

### CODE REVIEW

File: doc/contributing.rst [hunk 1]
```diff
@@ -290,7 +290,13 @@ First, you need to have a development environment to be able to build xarray
 Building the documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-So how do you build the docs? Navigate to your local
+So how do you build the docs? 
+
+Firstly, make sure you have ``sphinx`` and ``sphinx_rtd_theme`` installed::
+
+    conda install -c anaconda sphinx sphinx_rtd_theme
```
Comment (shoyer) [inline doc/contributing.rst]: This feels a little incongruent with the paragraph beginning with "First" under "Requirements" above

Comment (shoyer) [reply doc/contributing.rst]: Maybe: "In your development environment, install sphinx and sphinx_rtd_theme:"

**Comment by fmaussion** [comment]
Note that in the upcoming version of xarray the contributing page will be more prominent and visible in the docs: http://xarray.pydata.org/en/latest/contributing.html 

The two links in the README files link to the github page, which is less nice than the rendered readthedocs HTML

File: doc/contributing.rst (hunk 1 above)
Comment (raybellwaves) [reply doc/contributing.rst]: Agreed. I also dropped the question as i felt it was superfluous.

**Comment by raybellwaves** [comment]
@fmaussion Agreed. Changed the URsL to link to the latest version of the online docs.

**Comment by jhamman** [comment]
LGTM. Thanks @raybellwaves!

**Pull request merged**